hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -82,40 +150,41 @@
         }
       ]
     },
-    { name: "ES_CTRL",
-      desc: "ENTROPY_SRC control register",
-      swaccess: "rw",
-      hwaccess: "hro",
-      regwen:   "ES_REGEN",
-      fields: [
-        { bits: "0",
-          name: "INIT_ENT_SRC",
-          desc: "Setting this bit will initialize the entropy source function."
-        }
-      ]
-    },
     { name:     "ES_STATUS",
       desc:     "ENTROPY_SRC status register",
       swaccess: "ro",
-      hwaccess: "hrw",
+      hwaccess: "hwo",
       hwext: "true",
       fields: [
-        { bits: "0",
-          name: "ENTROPY_RDY",
-          desc: '''This is set when the entropy source FIFO has met or exceeded the ES_THRESH value. This status bit will
-                automatically clear once the FIFO depth drops below the threshold value. This bit will not set unless the
-                block is enabled.
-                '''
+        { bits: "2:0",
+          name: "FIFO0_DEPTH",
+          desc: "This is the depth of the lfsr entropy source FIFO."
+        }
+        { bits: "7:4",
+          name: "FIFO1_DEPTH",
+          desc: "This is the depth of the analog entropy source FIFO."
+        }
+        { bits: "10:8",
+          name: "FIFO2_DEPTH",
+          desc: "This is the depth of the software entropy delivery FIFO."
+        }
+        { bits: "14:12",
+          name: "FIFO3_DEPTH",
+          desc: "This is the depth of the hardware entropy delivery FIFO."
+        }
+        { bits: "31",
+          name: "DIAG",
+          desc: "This bit is for internal debug only."
         }
       ]
     },
     { name:     "ES_FDEPTHST",
```

Comment:
sorry i'm not too clear on how this one is different from the register immediately preceding.

Reply:
correct in that it is duplicated, but ES_FDEPTHST is the same as FIFO2 DEPTH. ES_FDEPTHST was kept for verification comparability.  Changed ES_STATUS to be ED_FIFO_STATUS, as it is a general debug register, getting all FIFO depth values with one read. Ok with this, or remove redundancy?

Reply:
I'm okay with either.  Just if it is a redundancy, i think it would be good to explicitly state in the documentation.

Reply:
ok, will make an explicit comment. thanks @tjaychen